packages/example-project/component-library/stencil.config.ts

```diff
@@ -66,8 +66,12 @@ export const config: Config = {
       hydrateModule: 'component-library/hydrate'
     }),
     vueOutputTarget({
```

Comment:
Should we have two Vue output target configs like we do for React?

Reply:
React and Vue projects are setup differently here. We have two output targets for React because we directly compile the components into their represented projects. With Vue we have setup an example project that just re-exports the components, and potentially more Vue related modules. That example project is imported by Nuxt and the Vite project. A bit different setup but it is actually how we see projects being organised.